languages/js/src/Polar.ts

```diff
@@ -254,6 +254,14 @@ export class Polar {
 
   /**
    * Register a JavaScript class for use in Polar policies.
+   *
+   * @param cls The class to register.
+   * @param params An optional object with extra parameters.
+   *
+   * Accepted extra parameters are:
```

Comment:
Can we make a follow up issue to go back and add types for these?

Reply:
Or an interface, whatever the right typescript-ism is

Reply:
Added it!